packages/react-sdk/src/index.tsx

```diff
@@ -198,22 +230,30 @@ export function useFeature(key: FeatureKey) {
   const feature = features[key];
   const enabled = feature?.isEnabled ?? false;
 
+  function sendCheckEvent() {
+    client
+      ?.sendCheckEvent({
+        key,
+        value: enabled,
+        version: feature?.targetingVersion,
+      })
+      .catch(() => {
+        // ignore
+      });
+  }
+
   return {
     isLoading,
     track,
     requestFeedback,
     get isEnabled() {
-      client
-        ?.sendCheckEvent({
-          key,
-          value: enabled,
-          version: feature?.targetingVersion,
-        })
-        .catch(() => {
-          // ignore
-        });
+      sendCheckEvent();
       return enabled;
     },
+    get config() {
+      sendCheckEvent();
+      return feature?.config ?? EMPTY_FEATURE_CONFIG;
```

Comment:
same problem here: someone could modify `EMPTY_FEATURE_CONFIG` and affect subsequent "default" returns